app/code/Magento/QuoteGraphQl/Model/Cart/QuoteAddressFactory.php

```diff
@@ -36,36 +38,55 @@ class QuoteAddressFactory
      */
     private $addressHelper;
 
+    /**
+     * @var CountryInformationAcquirerInterface
+     */
+    private $countryInformationAcquirer;
+
     /**
      * @param BaseQuoteAddressFactory $quoteAddressFactory
      * @param GetCustomerAddress $getCustomerAddress
      * @param AddressHelper $addressHelper
+     * @param CountryInformationAcquirerInterface|null $countryInformationAcquirer
      */
     public function __construct(
         BaseQuoteAddressFactory $quoteAddressFactory,
         GetCustomerAddress $getCustomerAddress,
-        AddressHelper $addressHelper
+        AddressHelper $addressHelper,
+        CountryInformationAcquirerInterface $countryInformationAcquirer = null
     ) {
         $this->quoteAddressFactory = $quoteAddressFactory;
         $this->getCustomerAddress = $getCustomerAddress;
         $this->addressHelper = $addressHelper;
+        $this->countryInformationAcquirer = $countryInformationAcquirer;
```

Comment:
We do not care about BC in GraphQL modules. Only the schema is out API.
https://github.com/magento/graphql-ce/wiki/Approach-to-Implementation#backward-compatibility-policy

Reply:
Ok.